lib/phoenix/controller/logger.ex

```diff
@@ -4,6 +4,10 @@ defmodule Phoenix.Controller.Logger do
   @behaviour Plug
   import Phoenix.Controller
 
+  alias Phoenix.Endpoint.Instrument
+
+  @filter_parameters Application.get_env(:phoenix, :filter_parameters)
```

Comment:
@chrismccord the downside of this solution is that, if you change the filter_parameters configuration, the option won't propagate unless you recompile all of Phoenix. So I would honestly move away from the module attribute because I don't think this cons > pros. But we already gave @stevedomin enough work, i will fix it after merging. :)